<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Инспектор аэропортов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Steel=rgb(0, 0, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="1">
     <widget class="QPushButton" name="pb_disconnect">
      <property name="text">
       <string>Разорвать соединение</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="lb_statusConnect">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dE_date">
      <property name="maximumDate">
       <date>
        <year>2017</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2016</year>
        <month>8</month>
        <day>15</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="lb_statistica">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Статистика полетов :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lb_date">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Дата :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lb_aeroport">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Аэропорт :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cb_listAiroport">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="14">
     <widget class="QTableView" name="tv_viewDateBase"/>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="rb_arrival">
      <property name="text">
       <string>Прилёт</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pb_receiverFlight">
      <property name="text">
       <string>Вывести рейсы</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pb_receive">
      <property name="text">
       <string>Получить статистику</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="rb_departure">
      <property name="text">
       <string>Вылет</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="pb_connect">
      <property name="text">
       <string>Соединение с базой данных</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionStatistica"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionStatistica">
   <property name="text">
    <string>Статистика</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
